crates/svm-rs/src/platform.rs

```diff
@@ -43,7 +43,10 @@ impl FromStr for Platform {
 }
 
 pub fn is_nixos() -> bool {
-    std::path::Path::new("/etc/NIXOS").exists()
+    cfg!(target_os = "linux")
```

Comment:
I think it is relatively reasonable to assume this and it would specifically fix the issue for the user in https://github.com/foundry-rs/foundry/issues/9682#issuecomment-2593001089